.travis.yml

```diff
@@ -1,11 +1,25 @@
 language: node_js
 
 node_js:
+  - 8
   - 6
 
+cache:
+  directories:
+    - ~/.npm
+
+notifications:
```

Comment:
There are also [HipChat notifications that we should implement later](https://docs.travis-ci.com/user/notifications/#Configuring-HipChat-notifications) (I just didn't have the HipChat authentication information)